tasmota/my_user_config.h

```diff
@@ -693,12 +693,12 @@
 //    #define INA3221_MAX_COUNT                    // change the number of devices to search for (default 4).
 //                                                 // Both settings together allow to limit searching for INA3221 to only a subset of addresses
 
-//  #define USE_RTC_CHIPS                          // Enable RTC chip support and NTP server - Select only one
-//    #define USE_DS3231                           // [I2cDriver26] Enable DS3231 RTC (I2C address 0x68) (+1k2 code)
-//    #define USE_BM8563                           // [I2cDriver59] Enable BM8563 RTC - found in M5Stack - support both I2C buses on ESP32 (I2C address 0x51) (+2.5k code)
-//    #define USE_PCF85363                         // [I2cDriver66] Enable PCF85363 RTC - found Shelly 3EM (I2C address 0x51) (+0k7 code)
+//  #define USE_RTC_CHIPS                        // Enable RTC chip support and NTP server - Select only one
+//    #define USE_DS3231                         // [I2cDriver26] Enable DS3231 RTC (I2C address 0x68) (+1k2 code)
+//    #define USE_BM8563                         // [I2cDriver59] Enable BM8563 RTC - found in M5Stack - support both I2C buses on ESP32 (I2C address 0x51) (+2.5k code)
+//    #define USE_PCF85363                       // [I2cDriver66] Enable PCF85363 RTC - found Shelly 3EM (I2C address 0x51) (+0k7 code)
 
-//  #define USE_DISPLAY                            // Add I2C Display Support (+2k code)
+//  #define USE_DISPLAY                          // Add I2C/TM1637/MAX7219 Display Support (+2k code)
```

Comment:
USE_DISPLAY has no specific relation to TM1637. No change needed

Reply:
When I compile with `#define USE_TM1637` and `#undef USE_DISPLAY` I get no dropdown for choosing the TM1637 module. When I `#define USE_DISPLAY` I can choose and use it.